modules/ROOT/pages/create-profiles.adoc

```diff
@@ -72,61 +72,87 @@ If you filter by categories, you can see the number of categories for each API,
 |
 Set Notifications
-|Enable or disable automatic notifications for this profile. The default is enabled. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
+|Enable or disable automatic notifications for this profile. The default setting is enabled for active profiles. Notifications are automatically disabled for draft profiles. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
 |Specify General Information 
 |Enter a profile name and description that helps others understand the kinds of governance rulesets included and the kinds of APIs validated in this profile.
 |Review
-|Review your profile and click *Create*.
+|Review your profile and click *Save as a draft* or *Save as active*:
+
+* Save a profile as draft to test profile settings and review conformance information before showing the conformance information for its targeted APIs outside of the draft profile view.
+* Save a profile as active to show the conformance information outside of the draft profile view.
+
+When you save a profile as either draft or active, its targeted APIs become governed.
 |
 |===
 
 == Update a Governance Profile Using the API Governance Console
 
-The API Governance console guides you through updating profiles. 
+The API Governance console UI guides you through updating profiles. 
 
-To update governance profile information using the API Governance console: 
+To update governance profile information using the console: 
 
-. In *API Governance*, select the console *Profiles* tab.
+. In the API Governance console *Profiles* tab, click *Draft* or *Active*.
 . For the profile you want to update, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line]) and then select *Edit*.
 . Navigate through the UI using the *Next* and *Previous* buttons and update the profile information as needed. For details on the options, see <<create-profile-in-console>>.
 . Review your changes and click *Update Profile*.
-
-NOTE: After you apply your edits, the total APIs are updated to indicate how many APIs
++
+After you apply your edits, the total APIs value refreshes to indicate how many APIs
 your governance profile includes based on your updated filter criteria.
 
-== Delete a Governance Profile Using the API Governance Console
+[[activate-draft-profile]]
+== Activate a Draft Governance Profile
 
-The API Governance console guides you through deleting profiles.
+To activate a draft profile: 
+
+. In the API Governance console *Profiles* tab, click *Draft* to show draft profiles.
+. Click *Activate* for the profile to activate.
++
+After you activate a profile, the APIs validated against the rulesets in that profile become governed APIs and their conformance information appears outside of the draft profile view.
```

Comment:
also notifications are sent for failed validations if they are enabled.

Reply:
Added